code/game/objects/items/explosives/mine.dm

```diff
@@ -202,6 +202,10 @@ Stepping directly on the mine will also blow it up
 		return
 
 	if(linked_mine && isliving(A))
+		var/mob/living/unlucky_person = A
+		// Don't trigger for dead xenos
+		if(isxeno(unlucky_person) && unlucky_person.stat == DEAD)
```

Comment:
why the isxeno check ?

Reply:
i don't wannna affect HvH balance where you can throw people at landmines to detonate them ? or should it also not trigger for anything dead in general?

Reply:
made it not trigger for anything dead.